scripts/annotate.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+from pycytominer.annotate import annotate
+
+
+def annotate_cells(aggregated_data, platemaps):
+    """Annoates
+
+    Parameters
+    ----------
+    aggregated_data : str
+        path pointing to aggregated dataset
+    platemaps : str
+        path pointing to platemaps
+
+    """
+
+    # input paths retrived by snakemake
+    annotated_outfile = str(snakemake.output)
+    platemap_df = pd.read_csv(barcodes)
```

Comment:
We had talked about defining `barcodes` separately,  right? Also, I thought you had said there were defined something like:

```python
barcodes = snakemake.barcodes
```

Reply:
Actually, the `barcodes` is from the function's paramter varaible: 

function variables were updated: 21305ee51b64193213e31d112de1f6e5e17eff03